src/App.js

```diff
@@ -1,26 +1,127 @@
 import React from 'react';
-import logo from './logo.svg';
+import FoodBox from './components/FoodBox'
+import AddFood from './components/AddFood'
+import foods from './foods.json';
+import FoodList from './components/FoodList'
 import './App.css';
+import 'bulma/css/bulma.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      foods: foods,
+      filteredFoods: foods,
+      formShow: false,
+      todayFoods: []
+    }
+  }
+  
+
+  handleAdd = (event) => {
+    event.preventDefault()
+    let foodName = event.currentTarget.foodName.value
+    let foodCalories  = event.currentTarget.foodCalories.value
+    let foodImage = event.currentTarget.foodImage.value
+
+    let newFood = [
```

Comment:
the variable name newFood implies this is one food element, but you are getting all the foods array (the new + all the other). Be mind with naming your variables as it really helps with readability.